Framework/Algorithms/src/CreateWorkspace.cpp

```diff
@@ -67,13 +67,6 @@ void CreateWorkspace::init() {
       std::make_unique<WorkspaceProperty<>>("ParentWorkspace", "", Direction::Input, PropertyMode::Optional),
       "Name of a parent workspace.");
   declareProperty(std::make_unique<ArrayProperty<double>>("Dx"), "X error values for workspace (optional).");
-  std::vector<std::string> propOptions{Parallel::toString(Parallel::StorageMode::Cloned),
-                                       Parallel::toString(Parallel::StorageMode::Distributed),
-                                       Parallel::toString(Parallel::StorageMode::MasterOnly)};
-  declareProperty("ParallelStorageMode", Parallel::toString(Parallel::StorageMode::Cloned),
```

Comment:
I'm concerned that removing this property could cause scripts/history to fail if anything used it. I wonder if we should keep it but log a warning that it is deprecated and will not be used?

Reply:
Yes, I agree